### DISCUSSION TITLE
Issue with widgets

### DISCUSSION THREAD

**Discussion by BlackLemmi** [baffler/Transparent-Twitch-Chat-Overlay#308] [Q&A]
Hi everybody. Love the module. But I tried using the alerts widget url from streamlabs to see my alerts on screen, but it doesn't work, I get an error file telling me the url is too long. It used to work but now it doesn't. I tried before with the alerts widget of twitch but that never worked, also because url was too long. So I now tried to "tinyurl" the streamlab widget url but still nothing. Can anyone help ? Or can the creator maybe change the module so that there is not a character limit to the url ? Thanx in advance :)

**Comment by sc4t** [comment]
I have the same problem problem in 2025..
